Userland/Libraries/LibGfx/Font/OpenType/Font.cpp

```diff
@@ -699,8 +699,15 @@ RefPtr<Gfx::Bitmap> Font::rasterize_glyph(u32 glyph_id, float x_scale, float y_s
     if (glyph_id >= glyph_count()) {
         glyph_id = 0;
     }
-    auto glyph_offset = m_loca->get_glyph_offset(glyph_id);
-    auto glyph = m_glyf->glyph(glyph_offset);
+
+    auto glyph_offset0 = m_loca->get_glyph_offset(glyph_id);
+    auto glyph_offset1 = m_loca->get_glyph_offset(glyph_id + 1);
```

Comment:
Seems like we can't know `glyph_id + 1` is a valid glyph index here. Should we tweak the bounds check above?

Reply:
There's a VERIFY in get_glyph_offset, the loca table is actually guaranteed to be one larger than the number of elements for this reason.

Reply:
Gotcha!